Pinta.Core/ImageFormats/OraFormat.cs

```diff
@@ -291,6 +287,18 @@ private BlendMode StandardToBlendMode (string mode)
 					return BlendMode.Screen;
 				case "svg:xor":
 					return BlendMode.Xor;
+                case "svg:hard-light":
+                    return BlendMode.HardLight;
+                case "svg:soft-light":
+                    return BlendMode.SoftLight;
+                case "svg:color":
+                    return BlendMode.Color;
+                case "svg:luminosity":
+                    return BlendMode.Luminosity;
+                case "svg:hue":
+                    return BlendMode.Hue;
+                case "svg:saturation":
+                    return BlendMode.Saturation;
 				default:
 					Console.WriteLine ("Unrecognized composite-op: {0}, using Normal.", mode);
```

Comment:
There have always been cases, like if we didn't support these .ora standard modes that I just added, or if you open an .ora from another application that had added its own extensions.  It might be nice to add a dialog, but it's not a new situation.  :)